### PR TITLE
#43 Test HelloWorld on Travis, without signing.

### PR DESCRIPTION
**Pull request by jperedadnr**
Added IosConfiguration (WIP)

### CODE REVIEW

File: src/main/java/com/gluonhq/substrate/model/IosConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.gluonhq.substrate.model;
+
+import java.util.List;
+
+public class IosConfiguration {
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/model/IosConfiguration.java]: Might be better to cal this IosSigningConfiguration or so? It only deals with a specific part of configuration.

Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/model/IosConfiguration.java]: It is for signing (that also includes setting given provisioning/identities), but also for including frameworks (iOS or third party)

File: src/main/java/com/gluonhq/substrate/model/IosConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    public void setSimulatorDevice(String simulatorDevice) {
+        this.simulatorDevice = simulatorDevice;
+    }
+
+    public String getSimulatorDevice() {
+        return simulatorDevice;
+    }
+
+    public List<String> getFrameworks() {
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/model/IosConfiguration.java]: I don't think this belongs here. 
What's the difference with additional link parameters?

File: src/main/java/com/gluonhq/substrate/model/IosConfiguration.java (hunk 1 above)
Comment (johanvos) [reply src/main/java/com/gluonhq/substrate/model/IosConfiguration.java]: As I stated in the other comment, I don't think frameworks need to be managed here. That is a generic component (libraries)

File: src/main/java/com/gluonhq/substrate/model/IosConfiguration.java (hunk 2 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/model/IosConfiguration.java]: You mean frameworks can be also added to Mac?

Comment (johanvos) [reply src/main/java/com/gluonhq/substrate/model/IosConfiguration.java]: even more general, why would we treat frameworks different from libraries? Is there another use for them apart then providing them to the linker?

**Pull request merged**